ui/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "vscode-azureextensionui",
     "author": "Microsoft Corporation",
-    "version": "0.33.9",
+    "version": "0.33.10",
```

Comment:
npm packages don't ship their dependencies, so publishing a new version is unnecessary

Reply:
I figured it processed the dependent package-lock files, component governance claimed ui was one of the components bringing this in.

Reply:
I don't see it in npm ls, will look at component governance again.